astropy/io/asdf/setup_package.py

```diff
@@ -0,0 +1,17 @@
+# Licensed under a 3-clause BSD style license
+import os
+
+def get_package_data():
```

Comment:
Right now it appears this is doing nothing since there is no `schemas` dir.  Is that right?  If so are you quite sure some schemas will be included in 3.0, or perhaps should this file be left out until that subsequent PR?

Reply:
Yes, that's true. I went back and forth on this and decided to leave it in since it will likely be required eventually, but it wouldn't be a big deal to remove it if you thought that was best.